pint/models/spindown.py

```diff
@@ -25,29 +25,28 @@ def __init__(self):
         # The number of terms in the taylor exapansion of spin freq (F0...FN)
         self.num_spin_terms = maxderivs
 
```

Comment:
But do we want to evaluate polynomials with a mix of float and longdouble? Seems simplest to keep all the F* parameters in the same representation.  F1+ would have to get promoted to longdouble anyway during the evaluation, wouldn't they?